caikit/runtime/grpc_server.py

```diff
@@ -250,15 +246,14 @@ def stop(self, grace_period_seconds: Union[float, int] = None):
             grace_period_seconds (Union[float, int]): Grace period for service shutdown.
                 Defaults to application config
         """
-        if not grace_period_seconds:
```

Comment:
Interesting. I'm assuming this is to allow an explicit `0` to be used?

Reply:
yes, I can be swayed either way tho?

Reply:
Given that this is just used to check the config value, I think this is a little better